frontend/src/app/main/ui/components/color_input.cljs

```diff
@@ -81,7 +94,14 @@
                (apply-value new-value)
                (update-input value)))))
 
-        ;; list-id (str "colors-" (uuid/next))
+        on-click
+        (mf/use-callback
+         (fn [event]
+           (let [target (dom/get-target event)]
+             (when (some? ref)
+               (let [current (mf/ref-val ref)]
+                 (when (and (some? current) (not (.contains current target)))
+                   (.blur current)))))))
```

Comment:
Missing ^js metadata on interop

Reply:
I'll better use the function in dom namespace